Makefile

```diff
@@ -78,3 +78,9 @@ db-down-test:
 	docker rm -f spi_test
 
 db-reset: db-down db-up migrate
+
+build-front-end:
+	docker run --rm -it -v $$PWD:/host -w /host --entrypoint yarn node:latest build
+
+serve-front-end:
+	docker run --rm -it -v $$PWD:/host -w /host --entrypoint yarn node:latest serve
```

Comment:
I think we should use the same pinned `node:15.8-alpine` version here so we see when/if we need to update.

Reply:
Fixed.